src/sentry/incidents/charts.py

```diff
@@ -203,6 +228,16 @@ def build_metric_alert_chart(
             user,
         ),
     }
+    if features.has(
```

Comment:
shouldn't this check if the `detection_type` on the `alert_rule` is dynamic as well?

Reply:
Yes you are correct :O